options/posix/include/mlibc/resolv_conf.hpp

```diff
@@ -10,11 +10,15 @@ namespace mlibc {
 struct nameserver_data {
 	nameserver_data()
 	: name(getAllocator()) {}
+	nameserver_data(const char *name_, int af_)
```

Comment:
This is a really minor nitpick, but we don't do this in the managarm code style, nor is it necessary.